src/pages/EmployeeProfilePage/EmployeeProfilePage.tsx

```diff
@@ -0,0 +1,147 @@
+import { useMutation, useQuery, useReactiveVar } from '@apollo/client';
+import { Box } from '@mui/material';
+import { ChangeEvent } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import EmployeeTabs from '../../components/EmployeeTabs';
```

Comment:
Just a suggestion Would be nice to set aliases for avoid import ... from '../../components/'
With aliases will look import ,,, from '@componts'

Reply:
will be done soon